axiom/nr_txn.c

```diff
@@ -3426,7 +3432,8 @@ static void nr_txn_add_log_event(nrtxn_t* txn,
     event_dropped = true;
   } else {
     /* event passed log level filter so add it */
-    e = log_event_create(log_level_name, log_message, timestamp, context_attributes, txn, app);
+    e = log_event_create(log_level_name, log_message, timestamp,
+                         context_attributes, txn, app);
```

Comment:
Stage the file and then run `git clang-format` to format only parts that were changed and stage again. This flow avoids reformatting the whole file which makes it harder to review.

Reply:
fixed in 36f308404aac646b13619b555d588463f5dc8fb1